src/ConnectionFactory.php

```diff
@@ -33,6 +33,7 @@ public static function getConnection(GenericBackendCredentials $credentials, boo
             'password' => $credentials->getSecret(),
             'port' => $credentials->getPort() === 0 ? 1025 : $credentials->getPort(),
             'dbname' => '',
+            'serverVersion' => '17.10',
```

Comment:
co to je? asi souhlasim, jen me zajima, kde jsme k tomu prisli

Reply:
viz commit message ;-)
> [add Teradata version to Connection to getSql from QueryBuilder](https://github.com/keboola/php-storage-driver-teradata/pull/44/commits/3e074072f9031e3f01ef15af12ac6ce50c332d38)